ui/src/organizations/containers/OrganizationsIndex.tsx

```diff
@@ -13,7 +13,7 @@ import {Button, IconFont, ComponentColor} from '@influxdata/clockface'
 import {OverlayTechnology} from 'src/clockface'
 
 // Actions
-import {createOrg, deleteOrg} from 'src/organizations/actions'
+import {createOrg, deleteOrg} from 'src/organizations/actions/orgs'
```

Comment:
nit.  why create another file called orgs under `organizations/actions`?

Reply:
the reducer was in a file called orgs so thought they should match?